express-post-json/server.js

```diff
@@ -0,0 +1,41 @@
+import express from 'express';
+
+const app = express();
+let nextId = 1;
+const grades = {};
+
+app.listen(8080, () => {
+  console.log('Listening on port 8080!');
+});
+
+app.get('/api/grades', (req, res) => {
+  const array = [];
+  for (const key in grades) {
+    array.push(grades[key]);
+  }
+  res.json(array);
+});
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  grades[nextId] = {
+    id: nextId,
+    name: req.body.name,
+    course: req.body.course,
+    score: req.body.score
+  };
+  res.status(201);
+  res.json(grades[nextId]);
+  nextId++;
```

Comment:
@codewithbingle Please make sure all work is completed before sending a response.

Reply:
Updated.